arangod/MMFiles/MMFilesAqlFunctions.cpp

```diff
@@ -415,7 +418,7 @@ void MMFilesAqlFunctions::registerResources() {
   TRI_ASSERT(functions != nullptr);
 
   // fulltext functions
-  functions->add({"FULLTEXT", ".h,.,.|.", false, true,
+  functions->add({"FULLTEXT", ".h,.,.|.", true, false,
                  false, true, &MMFilesAqlFunctions::Fulltext,
```

Comment:
mmmh. Now it can't throw anymore, that's probably what motivated the changes above.
But `FULLTEXT` is definitely not deterministic in the sense that query results can be cached etc.
So it should rather be `false, false`, or even better, reverted to `false, true`.